blog/dynatrace-datadog-instana-grafana-netdata.md

```diff
@@ -771,18 +847,78 @@ monthly GiB = rate * 86400 / 8 * 365 / 12 / 1024 / 1024
 
 | |Dynatrace|Datadog|Instana|Grafana|Netdata|
 |:----:|:----:|:----:|:----:|:----:|:----:|
-|rate (kbps)|12.0|25.2|16.1|13.0|0.3|
-|monthly (GiB)|3.68|7.73|4.94|3.99|0.09|
+|rate (kbps)|12.3|30.2|16.1|15.3|0.3|
+|monthly (GiB)|3.9|9.5|5.0|4.8|0.1|
+
+As shown, Netdata does not really use any internet traffic. Since Netdata does not push the samples and the logs to Netdata Cloud, the only bandwidth used is when users are viewing these data. We measured the bandwidth used when users view the dashboards via Netdata Cloud and we found that each Netdata uses on the average 15 kbps per viewer, for the time the viewers use a dashboard the node participates.
+
+## Pricing
+
+Assuming a node that:
+
+- runs 24x7
+- generates about 2 GiB of logs (or about 500k log entries) per month, retained for 30 days
+
+All prices are updated Mar 8, 2024, and refer to monthly billing.
+
+### Dynatrace
+
+|Features|Pricelist|Monthly Price/node
+|:---:|:---:|:---:|
+|Infrastructure monitoring|$0.04 per hour per node|$29.2
+|Application security|$0.018 per hour per node|$13.1
+|Logs Management and analytics|Ingest: $0.20/GiB<br/>Retain: $0.0007/GiB/day<br/>Query: $0.035/GiB|$1.0
+
```

Comment:
For Datadog and Dynatrace specifically, can we add the egress cost as additional infra costs?
And for Netdata add the storage costs (estimated) so we can do a real apple to apple comparison?